catroid/src/main/java/org/catrobat/catroid/CatroidApplication.java

```diff
@@ -41,6 +48,8 @@
 	public static final String OS_ARCH = System.getProperty("os.arch");
 
 	public static boolean parrotLibrariesLoaded = false;
+	public static String defaultSystemLanguage;
```

Comment:
initially 
this variable store the actual smartphone language, if we change the APP language, the variable 
`Locale.getDefault().getLanguage();` will store the language code on APP level not on OS level.
So, it is essential to store the language code of the smartphone when the user lunch the APP